physlr/physlr.py

```diff
@@ -1021,107 +1020,127 @@ def assign_read_molecule(mxs, moltomxs, mol_counts, bx):
         return "_" + str(random.choice(max_hits)), 0, 1
 
     @staticmethod
-    def determine_molecules_biconnected_components(g, u):
-        "Separate bi-connected components."
-        cut_vertices = set(nx.articulation_points(g.subgraph(g.neighbors(u))))
-        components = list(nx.connected_components(g.subgraph(set(g.neighbors(u)) - cut_vertices)))
+    def community_detection_biconnected_components(g, node_set):
```

Comment:
```suggestion
    def detect_communities_biconnected_components(g, node_set):
```
Methods should be named `verb_noun`, so `detect_communities_biconnected_components`.